cmd/node/main.go

```diff
@@ -1168,9 +1168,11 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	if check.IfNil(coreServiceContainer) {
```

Comment:
I think that lines 1168-1176 could be replaced with:
```
	if !check.IfNil(coreServiceContainer) && !check.IfNil(coreServiceContainer.Indexer()) {
		elasticIndexer = coreServiceContainer.Indexer()
		elasticIndexer.SetTxLogsProcessor(processComponents.TxLogsProcessor)
		processComponents.TxLogsProcessor.EnableLogToBeSavedInCache()
	}

```

Reply:
done